app/src/main/java/app/revanced/manager/ui/viewmodel/PatcherViewModel.kt

```diff
@@ -243,6 +243,10 @@ class PatcherViewModel(
         tempDir.deleteRecursively()
     }
 
+    fun isDeviceRooted() = System.getenv("PATH")?.split(":")?.any { path ->
+        File(path, "su").canExecute()
+    } ?: false
```

Comment:
This does not belong here. Move it to `RootInstaller`.

Reply:
Done